app/src/main/java/ar/com/wolox/android/training/ui/training/adapter/NewsServiceAdapter.kt

```diff
@@ -24,4 +25,22 @@ class NewsServiceAdapter @Inject constructor(private val retrofitServices: Retro
                 }
         )
     }
+
+    fun modifyNews(positionId: Int, body: NewsItem, listener: NewsPutServiceAdapterListener) {
```

Comment:
Remember it's not a `positionId` for business logic, and even less for the adapter that it's just to connect to the service. It's just the id. The same happens with the `body`, it's a news. Also, I think since you're already receiving the news, it'll contains the id, so it's unnecessary to receive the id.